### ISSUE TITLE
Use handsontable for the data preview

### ISSUE THREAD

**Issue by alex-r-bigelow** [Kitware/candela#474]
[handsontable](https://handsontable.com/) should give us excel-style interactions for free (filtering, sorting, ... editing?). And I imagine it will be a bit more performant than the table code we have in Resonant Lab now...

**Comment by ronichoudhury** [comment]
Interesting. Do you know why they say "free trial" on the site when it's
already MIT-licensed and released on GitHub?

roni

**Comment by alex-r-bigelow** [comment]
They have some pro features that they want you to pay for... but nothing we'd really need. I think someone in their marketing department sneakily decided to hide that you can simply `npm install handsontable` for free, and all the non-pro features are still available.

**Comment by alex-r-bigelow** [comment]
Of course, I'm happy to worry about this when / after I do the matching interface overhaul
